web/src/Index.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { render } from 'react-dom';
-import './index.css';
+import './Index.css';
```

Comment:
This was made in order to get the local server running on my computer. I'm not sure if this will cause errors for others

Reply:
I had that same issue! Thanks for updating here.